docker-compose.yml

```diff
@@ -0,0 +1,134 @@
+version: '3.7'
+services:
+  zookeeper-1:
+    image: 'confluentinc/cp-zookeeper:${CONFLUENT_PLATFORM_VERSION:-5.5.0}'
```

Comment:
is `CONFLUENT_PLATFORM_VERSION` intended to be added as part of the ci ENV variables or Makefile?
if not, we can ✂️

Reply:
nvm, not sure why github didn't show the entire diff, I see you're using it later.